medicines/api/Makefile

```diff
@@ -0,0 +1,26 @@
+.PHONY: default
+default: ## Run locally
+	export $$(cat .env .env.overrides 2> /dev/null | xargs) && cargo run
+
+.PHONY: test
+test: ## Run tests [TEST=test_name (optional)]
+	export $$(cat .env .env.overrides 2> /dev/null | xargs) && cargo test $$TEST
+
+.PHONY: get-env
+get-env: ## Gets the environment variables from azure keyvault into .env file
+	az keyvault secret show \
+	  --vault-name mhra-non-prod-02 \
+	  --name api-env \
```

Comment:
Will this be the envs for the dev environment or nonprod? Would be good if it was named more specifically to remove doubt (since we're using the same `mhra-non-prod-02` vault for both)

Reply:
I've moved the dev environments into a vault called `mhra-dev`.